libraries/AP_Landing/AP_Landing.cpp

```diff
@@ -145,6 +145,15 @@ const AP_Param::GroupInfo AP_Landing::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("OPTIONS", 16, AP_Landing, _options, 0),
 
+    // @Param: FLARE_AIM
+    // @DisplayName: Flare aim point adjustment percentage.
+    // @Description: This parameter controls how much the aim point is moved to allow for the time spent in the flare manoeuvre. When set to 100% the aim point is adjusted on the assumption that the flare sink rate controller instantly achieves the sink rate set by TECS_LAND_SINK. when set to 0%, no aim point adjustment is made. If the plane consistently touches down short of the aim point reduce the parameter and vice verse.
+    // @Range: 0 100
+    // @Units: %
+    // @Increment: 1
+    // @User: Advanced
+    AP_GROUPINFO("FLARE_AIM", 17, AP_Landing, flare_effectivness_pct, 50),
```

Comment:
This is somehow a factor of TECS response time I think? I wonder if we could calculate a value.

Reply:
Unfortunately  its a function of aircraft lift curve slope, approach speed, glide slope, pitch loop tuning, TECS tuning, etc. The reason I added this parameter is that after the aircraft is tuned and a standardised glide slope and airspeed is determined, this parameter can then be used to fine tune the landing accuracy,